core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-add-variable/dot-add-variable.component.ts

```diff
@@ -1,38 +1,50 @@
-import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { Component, OnInit, inject } from '@angular/core';
 
 import { DynamicDialogConfig, DynamicDialogRef } from 'primeng/dynamicdialog';
 
 import { map } from 'rxjs/operators';
 
 import { DotAddVariableStore } from '@dotcms/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-add-variable/store/dot-add-variable.store';
+import { DotMessagePipe } from '@dotcms/ui';
 
-import { FilteredFieldTypes } from './dot-add-variable.models';
+import { DotVariableContent, DotVariableList, FilteredFieldTypes } from './dot-add-variable.models';
 
 @Component({
     selector: 'dot-add-variable',
     templateUrl: './dot-add-variable.component.html',
     styleUrls: ['./dot-add-variable.component.scss'],
-    providers: [DotAddVariableStore]
+    providers: [DotAddVariableStore, DotMessagePipe]
 })
 export class DotAddVariableComponent implements OnInit {
-    vm$ = this.store.vm$.pipe(
+    private readonly dotMessage = inject(DotMessagePipe);
+    private readonly store = inject(DotAddVariableStore);
+    private readonly config = inject(DynamicDialogConfig);
+    private readonly ref = inject(DynamicDialogRef);
+
+    vm$: Observable<DotVariableList> = this.store.vm$.pipe(
         map((res) => {
-            const variables = res.variables.filter(
-                (variable) =>
-                    variable.fieldType !== FilteredFieldTypes.Column &&
-                    variable.fieldType !== FilteredFieldTypes.Row
-            );
+            const variables: DotVariableContent[] = res.variables
+                .filter(
+                    (variable) =>
+                        variable.fieldType !== FilteredFieldTypes.Column &&
+                        variable.fieldType !== FilteredFieldTypes.Row
+                )
+                .map((variable) => ({
+                    name: variable.name,
+                    variable: variable.variable,
+                    fieldTypeLabel: variable.fieldTypeLabel
+                }));
+            variables.push({
+                name: this.dotMessage.transform('Content-Identifier-value'),
```

Comment:
Use the `DotMessageService` it has a`get` method for this

Reply:
`DotMessagePipe` is a wrapper of `DotMessageService`

Reply:
Yep, DotMessagePipe is a wrapper, but for Separation of Concerns, i change it to directly MessageService (In the new version of PR)